documentation/src/main/asciidoc/topics/con_xsite_backup_strategies.adoc

```diff
@@ -0,0 +1,34 @@
+[id='xsite_backups-{context}']
+= Backup Strategies
+{brandname} clusters can use different strategies for backing up data to remote sites.
+
+{brandname} replicates across sites at the same time that writes to local caches occur. For example, if a client writes "k1" to **LON**, {brandname} backs up "k1" to **NYC** at the same time.
+
+.Synchronous backups
+When {brandname} applies changes to the remote site, it waits until the operation completes.
+
+The synchronous strategy lets you define policies that control how {brandname} handles writes if backup operations fail.
+
+.{brandname} failure policies
+|===
+| Policy | If backup operations fail...
+
+| `IGNORE`
+| {brandname} ignores exceptions and writes to the local cache.
+
+| `WARN`
+| {brandname} logs exceptions and writes to the local cache. This is the default failure policy.
+
+| `FAIL`
+| {brandname} throws exceptions and aborts writes to the local cache.
```

Comment:
if it can to abort :)

Reply:
{brandname} throws exceptions and attempts to abort writes to the local cache. In some cases, local writes still succeed if the backup writes fail.